packages/nc-gui-v2/components/virtual-cell/BelongsTo.vue

```diff
@@ -315,7 +315,7 @@ export default {
     <!--    <template v-if="!isForm"> -->
     <div class="chips d-flex align-center img-container flex-grow-1 hm-items">
       <template v-if="value || localState">
-        <ItemChip :active="active" :item="value" :value="value[primaryValueProp]" />
+        <ItemChip :active="active" :item="value" :model-value="value[primaryValueProp]" />
```

Comment:
should be v-model, shouldn't it? Or do you not want the value updated?

Reply:
just want to pass the primary value to display... edit will open an expanded form so may be I will switch it back to value, 2 way binding is not necessary here